src/ja/server/database/sql/database.py

```diff
@@ -160,6 +164,7 @@ def __init__(self, host: str = None, user: str = None, password: str = None):
             self.engine = create_engine(_conn)
             self.scoped = scoped_session(sessionmaker(self.engine))
             SQLDatabase._metadata.create_all(self.engine)
+            logger.info("connection to real database")
```

Comment:
Maybe print the name of the database and the IP? (username and password are confidential so not a good fit).

Reply:
but the database will be always called jobadder or?

Reply:
I guess this is something johannes should change in his PR. But otherwise, it is still good to see that the values are read from the config file correctly.

Reply:
yes he changed it in his pr, should I then wait for it to be merged before solving this one.